scripts/docker-run-rootfs.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_NAME="openwrt-rootfs:latest"
+ROOTFS_PATH="./bin/targets/x86/64/openwrt-x86-64-generic-rootfs.tar.gz"
+
+[ -f "$ROOTFS_PATH" ] || echo "Couldn't find rootfs at $ROOTFS_PATH"
+
+docker import \
+  --change "EXPOSE 80 443 22" \
+  --change "USER root" \
+  --change 'CMD ["/sbin/init"]' \
+  $ROOTFS_PATH \
+  "$IMAGE_NAME"
+echo "[*] Imported: $ROOTFS_PATH"
+
+docker network create \
+  --driver=bridge \
+  --subnet=192.168.1.0/24 \
+  --ip-range=192.168.1.0/24 \
+  --gateway=192.168.1.2 \
+  openwrt-lan  
+echo "[*] Created openwrt-lan network "
+
+
+docker create -it --net openwrt-lan --ip="192.168.1.1" --name openwrt-docker $@ "$IMAGE_NAME"
```

Comment:
Shellcheck says to quote "$@"

Reply:
Done - thanks for the shellcheck advise!